### DISCUSSION TITLE
Backup and restore

### DISCUSSION THREAD

**Discussion by Zaxx69** [sayaka-sh/spmp#94] [Q&A] [answered]
How can we backup our songs and restore it again

**Comment by sayaka-sh** [comment]
What do you want to backup? The list of library songs?

**Reply by Zaxx69** [reply]
Yeah bro and playlist too

**Reply by sayaka-sh** [reply, marked as answer]
There's no functionality for that at the moment, as the library page (and especially the local songs functionality) is still very much a work-in-progress. Feel free to create a feature request though, as I can definitely implement this at some point.

**Comment by rutipsi** [comment]
I don't understand. Isn't YouTube backing up your playlist?
